libs/core/models/anime.ts

```diff
@@ -0,0 +1,50 @@
+
+import { AnimeStatus } from './anime-status';
+
+import { AnimeType } from './anime-type';
+
+import { DateTimeRange } from './datetime-range';
```

Comment:
Please keep an eye on imports. Make sure there are no unnecessary empty lines
```suggestion
import { AnimeStatus } from './anime-status';
import { AnimeType } from './anime-type';
import { DateTimeRange } from './datetime-range';
```